doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md

```diff
@@ -0,0 +1,354 @@
+---
+layout: page
+title: Cassandra log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender
+sort_idx: 10
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The Cassandra log appender is responsible for transferring logs from the Operations server to the Cassandra database.
+
+# Creating Cassandra log appender with Admin UI
+
+The easiest way to create a Cassandra log appender for your application is by using Admin UI.
+
+To create a log appender do the following:
+
+1. In the <b>Log appenders</b> window, click <b>Add log appender</b>.
+2. Enter the log appender name and description, select the minimum and maximum supported schema version, select necessary log metadata fields.
+3. Set the log appender type to <i>Cassandra</i>. <br/>
+<img src="attach/create-cassandra-log-appender-admin-ui.png">
+4. Fill in the Cassandra log appender [Configuration](#configuration) form. <br/>
+![](attach/cassandra-log-appender-config1.png) <br/>
+![](attach/cassandra-log-appender-config2.png) <br/>
+![](attach/cassandra-log-appender-config3.png) <br/>
+
+5. Click <b>Add</b>.
+
+
+# Creating Cassandra log appender with Admin REST API
+
+It is also possible to create a Cassandra log appender instance by using [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs #TODO).
```

Comment:
I will add correct link after resolving this ticket: http://jira.kaaproject.org/browse/KAA-1108.